supertokens_python/recipe/thirdpartypasswordless/recipe.py

```diff
@@ -14,14 +14,21 @@
 from __future__ import annotations
```

Comment:
smtp service is not done for this.

Reply:
Updated `tppless/emaildelivery/service/smtp/__init__.py`